config/game_options.txt

```diff
@@ -37,8 +37,8 @@ EMOJIS
 ## To speed things up make the number negative, to slow things down, make the number positive.
 
 ## These modify the run/walk speed of all mobs before the mob-specific modifiers are applied.
-RUN_DELAY 1
-WALK_DELAY 4
+RUN_DELAY 2
+WALK_DELAY 5
```

Comment:
These should be left to their default values

Reply:
:thinking:

Reply:
but tg is sooo faaaast